css-ui-4/Overview.bs

```diff
@@ -2199,6 +2212,61 @@ that are typical of the element whose appearance it acquires.
 	is also not changed by the 'appearance' property.
 </div>
 
+<h3 id="input-security">
+Obscuring sensitive input: the 'input-security' property</h3>
+
+<pre class=propdef>
+Name: input-security
+Value: auto | none
+Initial: ''input-security/auto''
+Applies to: [=sensitive text inputs=]
+Inherited: no
+Percentages: N/A
+Computed Value: as specified
+Canonical order: per grammar
+Animation type: by computed value type
+</pre>
+
+For the purpose of this specification,
+a <dfn>sensitive text input</dfn> is
+a <a spec=html>mutable</a> form control with textual content whose
+purpose is to accept sensitive input,
+such as <{input/type/password|&lt;input type=password&gt;}>.
```

Comment:
```suggestion
a <dfn export>sensitive text input</dfn> is
a text input whose purpose is to accept sensitive input,
as defined by the host language.
For example, in HTML <{input/type/password|&lt;input type=password&gt;}> is a [=sensitive text input=].
```

Reply:
(You don't want to say "mutable", readonly inputs aren't mutable but they still hide their text. And generally we just want to be clear this is host-defined.)